src/cli.yaml

```diff
@@ -278,7 +279,7 @@ subcommands:
                     help: Standard deviation of expected insert size. Defines search space of the most likely overlap. (Required for calculating overlapping consensus only)
         - bam:
             about: |
-              Tool to remove PCR duplicates from BAM file.
+              Tool to predict maximum likelihood fragment sequence from BAM file.
```

Comment:
Maybe extend to:

    Tool to merge sets of PCR duplicate reads in a BAM file into one maximum likelihood fragment sequence each.